addons/airway/functions/fnc_treatmentAdvanced_CancelRecoveryPositionLocal.sqf

```diff
@@ -20,6 +20,7 @@ params ["_medic", "_patient"];
 
 _patient setVariable [QGVAR(recovery), false, true];
 _patient setVariable [QGVAR(overstretch), false, true];
+_patient call FUNC(handlePuking);
```

Comment:
not really needed imo because that fnc is PFH that execute code every X seconds and it checks if unit is in recovery

Reply:
I tested it and it did not work with 100% chance for occlusion after canceling. Like it never reinitiated the handle puking and therefore did never happen again.